lib/search_results/view/search_results_page.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:pattern_repository/pattern_repository.dart';
+import 'package:prechac_this/app/app_router.dart';
+import '../search_results.dart';
+
+class SearchResultsPage extends StatelessWidget {
+  SearchResultsPage({
+    Key? key,
+    required this.searchParameters,
+  }) : super(key: key);
+
+  static const routeName = 'results';
+  static String routeNameWithParameters(SearchParameters parameters) {
+    return Uri(
+      pathSegments: [routeName],
+      queryParameters: parameters.toQueryParameters(),
+    ).toString();
+  }
+
+  final SearchParameters searchParameters;
+
+  @override
+  Widget build(BuildContext context) {
+    return RepositoryProvider(
+      create: (_) => PatternRepository(),
+      child: BlocProvider(
+        create: (_) => SearchBloc(),
+        child: SearchResultsView(),
+      ),
```

Comment:
Remove provider for now:

```suggestion
    return SearchResultsView(),
```